manifest.json

```diff
@@ -112,6 +112,20 @@
           "js": ["app/angular.audio.js"]
         }
       },
+      "angularRecorder": {
+        "version": "1.4.1",
+        "downloadFormat": "zip",
+        "url": "https://github.com/logbon72/angular-recorder/archive/1.4.1.zip",
+        "rootDirPrefix": "ngAudioRecorder-",
+        "targetDirPrefix": "angular-recorder-"
```

Comment:
Do you need to specify bundle or files here, when I try running the local server it seems it can't find this library

Reply:
Nvm about this